apps/pom.xml

```diff
@@ -498,7 +498,7 @@ checkJavaFormat() {
   cd apps
 
   set +e
-  mvn com.coveo:fmt-maven-plugin:check >/dev/null 2>&amp;1
+  mvn com.coveo:fmt-maven-plugin:check &gt;/dev/null 2&gt;&amp;1
```

Comment:
GitHub's web editor for conflict resolution did bad things.

Reply:
It has not, i tried running it locally with the replacements and it doesnt work. It actually has an error in the shell itself: `parse error near `;&'`

Reply:
The escapes look right to me.  At least in master several other lines are also escaped.

To make it all more readable you might want to just wrap the whole block as CDATA so you can avoid having the escapes at all while remaining as valid XML.

Something like

```
<pre-commit><![CDATA[
```

and

```
]]></pre-commit>
```

Reply:
> To make it all more readable you might want to just wrap the whole block as CDATA so you can avoid having the escapes at all while remaining as valid XML.

Implemented as described. After doing a _little_ testing locally, I see that between master and this branch at a4e357d, the resultant `.git/hooks/pre-commit` installed yields the same md5sum: `708eea275a7c6aa5f1a28c678fbfdbaf`.